numba/targets/boxing.py

```diff
@@ -338,12 +338,14 @@ def box_smart_array(typ, value, c):
     with c.builder.if_else(cgutils.is_not_null(c.builder, obj)) as (has_parent, otherwise):
         with has_parent:
             c.pyapi.incref(obj)
-            retn = c.pyapi.call_method(obj, "host_changed")
+            host = c.pyapi.string_from_constant_string('host')
```

Comment:
Are there any tests anywhere for the pyapi module ? The error went indeed unnoticed, because we only called this method without arguments.